tests/integration_tests/default_value/main.go

```diff
@@ -0,0 +1,734 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"database/sql"
+	"flag"
+	"fmt"
+	"os"
+	"reflect"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/tests/integration_tests/util"
+	"go.uber.org/zap"
+)
+
+var finishIdx int
+
+func main() {
+	cfg := util.NewConfig()
+	err := cfg.Parse(os.Args[1:])
+	switch errors.Cause(err) {
+	case nil:
+	case flag.ErrHelp:
+		os.Exit(0)
+	default:
+		log.S().Errorf("parse cmd flags err %s\n", err)
+		os.Exit(2)
+	}
+
+	sourceDB0, err := util.CreateDB(cfg.SourceDBCfg[0])
+	if err != nil {
+		log.S().Fatal(err)
+	}
+	defer func() {
+		if err := util.CloseDB(sourceDB0); err != nil {
+			log.S().Errorf("Failed to close source database: %s\n", err)
+		}
+	}()
+	sourceDB1, err := util.CreateDB(cfg.SourceDBCfg[1])
+	if err != nil {
+		log.S().Fatal(err)
+	}
+	defer func() {
+		if err := util.CloseDB(sourceDB1); err != nil {
+			log.S().Errorf("Failed to close source database: %s\n", err)
+		}
+	}()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	go switchAsyncCommit(ctx, sourceDB0)
+	util.MustExec(sourceDB0, "create database mark;")
+	testGetZeroValue([]*sql.DB{sourceDB0, sourceDB1})
+	testGetDefaultValue([]*sql.DB{sourceDB0, sourceDB1})
+	util.MustExec(sourceDB0, "create table mark.finish_mark(a int primary key);")
+}
+
+// for every DDL, run the DDL continuously, and one goroutine for one TiDB instance to do some DML op
+func testGetDefaultValue(srcs []*sql.DB) {
+	runTime := time.Second * 60
+	start := time.Now()
+	defer func() {
+		log.S().Infof("testGetDefaultValue take %v", time.Since(start))
+	}()
+
+	var total int
+	for i, ddlFunc := range []func(context.Context, *sql.DB){
+		addDropColumnDDL, modifyColumnDefaultValueDDL1, modifyColumnDefaultValueDDL2,
+	} {
+		testName := getFunctionName(ddlFunc)
+		log.S().Info("running ddl test: ", i, " ", testName)
+
+		var wg sync.WaitGroup
+		ctx, _ := context.WithTimeout(context.Background(), runTime)
+
+		for idx, src := range srcs {
+			wg.Add(1)
+			go func(i int, s *sql.DB) {
+				dml(ctx, s, testName, i, nil)
+				wg.Done()
+			}(idx, src)
+		}
+
+		time.Sleep(time.Millisecond)
+
+		wg.Add(1)
+		go func() {
+			ddlFunc(ctx, srcs[0])
+			wg.Done()
+		}()
+
+		wg.Wait()
+
+		util.MustExec(srcs[0], fmt.Sprintf("create table mark.finish_mark_%d(a int primary key);", i+finishIdx))
+		total++
+	}
+
+	finishIdx += total
+}
+
+func switchAsyncCommit(ctx context.Context, db *sql.DB) {
+	ticker := time.NewTicker(5 * time.Second)
+	defer ticker.Stop()
+	enabled := false
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-ticker.C:
+			if enabled {
+				util.MustExec(db, "set global tidb_enable_async_commit = off")
+			} else {
+				util.MustExec(db, "set global tidb_enable_async_commit = on")
+			}
```

Comment:
Is it more explicit to set `on` when enabled is true?

Reply:
Besides we need to check the behavior of global variable, it seems set global variable `only apply to new connection sessions with TiDB`. https://docs.pingcap.com/tidb/stable/system-variables/#system-variables

Reply:
Get the point. I will remove the this global vasriable, because it seems has no effect on this test.